_examples/federation/readme.md

```diff
@@ -1,3 +1,14 @@
 ### Federation
 
 [Read the docs](https://gqlgen.com/recipes/federation/)
+
+## Testing
```

Comment:
I am a huge fan of making it easier for people besides me to debug through this 😆